command/server/server_command.go

```diff
@@ -143,8 +143,15 @@ func (c *ServerCommand) DefineFlags() {
 		FlagOptional: true,
 	}
 
-	c.FlagMap["max-peers"] = helper.FlagDescriptor{
-		Description: fmt.Sprintf("Sets the client's max peer count. Default: %d", helper.DefaultConfig().Network.MaxPeers),
+	c.FlagMap["max-inbound-peers"] = helper.FlagDescriptor{
+		Description: "Sets the client's maximum number of inbound peers allowded. Default: 40",
```

Comment:
Is it possible to give default value from helper.DefaultConfig().Network so that we can follow up the change of default config easily?

Reply:
Done, commit-[link](https://github.com/0xPolygon/polygon-edge/pull/343/commits/6514771aac425fde81aa4f51c72c40e41a696a7b)